03.asynchronous/callback.js

```diff
@@ -0,0 +1,60 @@
+import sqlite3 from "sqlite3";
+import timers from "timers/promises";
+
+//エラーなし
```

Comment:
`//` のあとにはスペースを入れましょう。30 行目についても同様です。また、変数 `db` を定義する処理の直前にこのコメントがありますが、変数 `db` は、エラーなしでもエラーありでも共通して使われる変数なので、このコメントの位置としては不適当だと思います。

```suggestion
// エラーなし
```

Reply:
ESLint・Prettierが自動で直してくれると思い込んでおり、気にしていませんでした💦
コメントの書き方と位置を修正しました。https://github.com/SuzukaHori/js-practices/pull/3/commits/a7a9fa1fae6e92b51f0f8e5c39d6bf70bd72dea2